test/e2e/e2e_test.go

```diff
@@ -642,6 +642,25 @@ func checkMetricsCollection(t *testing.T, promClient prometheusv1.API, namespace
 	}
 }
 
+func checkSignerCAMetrics(t *testing.T, promClient prometheusv1.API) {
+	timeout := 60 * time.Second
+	err := wait.PollImmediate(10*time.Second, timeout, func() (bool, error) {
+		results, warnings, err := promClient.Query(context.Background(), `signer_ca_expiry_time`, time.Now())
+		if err != nil {
+			t.Errorf("failed to execute prometheus query: %v", err)
+			return false, nil
+		}
+		if len(warnings) > 0 {
+			t.Logf("prometheus query emitted warnings: %v", warnings)
+		}
+
+		return len(results.String()) != 0, nil
```

Comment:
Please confirm that the values of `signer_ca_expiry_time` and `signer_ca_time_left` are as expected (i.e retrieve the signing secret in the test and check against its expiry).

Reply:
updated unit test to check both the values are as expected.